lisa/sut_orchestrator/util/device_pool.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, List, Optional
+
+from lisa.sut_orchestrator.util.schema import HostDevicePoolSchema, HostDevicePoolType
+from lisa.util import LisaException, SkippedException
+
+
+class BaseDevicePoolImpl:
+    def _create_device_pool(
+        self,
+        pool_type: HostDevicePoolType,
+        vendor_id: str,
+        device_id: str,
+    ) -> None:
+        raise NotImplementedError("'_create_device_pool' is not implemented")
```

Comment:
`raise NotImplementedError` is enough. The call trace has more details, and you don't need to maintain the string.

Reply:
Sure Chi, will remove the string msg.